e2e/src/bin/historic_prices.rs

```diff
@@ -1,10 +1,13 @@
 use anyhow::Result;
 use core::{history::Settlement, models::BatchId};
 use e2e::cmd;
-use pricegraph::{Element, Pricegraph, TokenPair};
+use pricegraph::{Element, Pricegraph, TokenId};
 use std::{fs::File, io::Write, path::PathBuf};
 use structopt::StructOpt;
 
+/// Threshold at which a price estimate is considered "bad"; currently 10%.
+const BAD_ESTIMATE_THRESHOLD: f64 = 0.1;
```

Comment:
In practice this is 10000 for the current solver, right?

Reply:
Yeah, this is just to get an idea for "close-ish" estimates.